sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 0.3.0-beta.3 (Unreleased)
+- Removing gltf support, given that it actually doesn't really work on the service: it's rejects with a 400 error
```

Comment:
This would actually go into the 0.3.0-beta.2 release as it is marked as "Unreleased", which indicates that it would be the next one to be released.